src/app/_components/RevokeAttestation.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from "@/components/ui/button";
+import useRevokeAttestation from "@/hooks/useRevokeOnChainAttestation";
+
+const RevokeAttestationPage = ({
+  uid,
+  schemaId,
+}: {
+  uid: string;
+  schemaId: string;
+}) => {
+  const { revokeAttestation, loading, error, success } = useRevokeAttestation();
+
+  const handleRevoke = () => {
+    revokeAttestation(uid, schemaId);
+  };
+
+  return (
+    <div>
+      <Button
+        variant={"outline"}
+        className="bg-red-500 text-white"
+        onClick={handleRevoke}
+        disabled={loading}
```

Comment:
`#should` also be disabled when it's already revoked

Reply:
yes , adding this

Reply:
done